src/platform/plugins/shared/ai_assistant_management/selection/public/routes/components/ai_assistant_selection_page.tsx

```diff
@@ -33,14 +33,19 @@ export function AiAssistantSelectionPage() {
     buildFlavor,
     kibanaBranch,
     securityAIAssistantEnabled,
+    observabilityAIAssistantEnabled,
   } = useAppContext();
   const aiAssistantManagementSelection =
     capabilities.management.kibana.aiAssistantManagementSelection;
-  const observabilityAIAssistantEnabled = capabilities.observabilityAIAssistant?.show;
+  const observabilityAIAssistantCapabilityEnabled = capabilities.observabilityAIAssistant?.show;
 
   const observabilityDoc = getDocLinks({ buildFlavor, kibanaBranch }).observability.aiAssistant;
   const securityDoc = getDocLinks({ buildFlavor, kibanaBranch }).securitySolution.aiAssistant;
   const isSecurityAIAssistantEnabled = securityAIAssistantEnabled && aiAssistantManagementSelection;
+  const isObservabilityAIAssistantEnabled =
+    observabilityAIAssistantCapabilityEnabled &&
+    observabilityAIAssistantEnabled &&
+    aiAssistantManagementSelection;
```

Comment:
Do we need this if the app is already disabled? They can't get to this page anyway?